Update view-service-health.md (#5510)
MicrosoftDocs/microsoft-365-docs: microsoft-365/enterprise/view-service-health.md

    > [!NOTE]
    > People who are assigned the global admin or service support admin role can view service health. To allow Exchange, SharePoint, and Skype for Business admins to view service health, they must also be assigned the Service admin role. For more information about roles that can view service health, see [About admin roles](../admin/add-users/about-admin-roles.md?preserve-view=true&view=o365-worldwide#commonly-used-microsoft-365-admin-center-roles).

2. To view service health, in the admin center, go to **Health** > **Service health**, or select the **Service health** card on the **Home dashboard**. The dashboard card indicates whether there is an active service issue and links to the detailed **Service health** page.

3. On the **Service health** page, the health state of each cloud service is shown in a table format.

   ![View of current issues in service health](../media/service-health-all-services.png)

> The **All services** tab (the default view) shows all services, their current health state, and any active incidents or advisories. An icon and status in the **Health** column indicates the state of each service.

Reviewer: ```suggestion
The **All services** tab (the default view) shows all services, their current health state, and any active incidents or advisories. An icon and status in the **Health** column indicate the state of each service.
```

> The **All services** tab (the default view) shows all services, their current health state, and any active incidents or advisories. An icon and status in the **Health** column indicates the state of each service.
> 
> If there is an active incidnet or advisory for a service they will be listed directly under the service name in a nested table. You can collapse the nested table to hide the incidents or advisories in this view by clicking on the cheveron icon to the left of the service name.   

Reviewer: ```suggestion
The **All services** tab (the default view) shows all services, their current health state, and any active incidents or advisories. An icon and status in the **Health** column indicate the state of each service.

If there is an active incident or advisory for a service they will be listed directly under the service name in a nested table. You can collapse the nested table to hide the incidents or advisories in this view by clicking on the chevron icon to the left of the service name.   
```

> To filter your view to only show all the active incidents, select the **Incidents** tab at the top of the page. Selecting the **Advisories** tab will only show all the active  advisories posted.

Reviewer: ```suggestion
To filter your view to only show all the active incidents, select the **Incidents** tab at the top of the page. Selecting the **Advisories** tab will only show all the active advisories posted.
```

> The **History** tab shows all incidents and advisories that have been resolved with in the last 7 or 30 days.

Reviewer: ```suggestion
The **History** tab shows all incidents and advisories that have been resolved within the last seven or 30 days.
```

> If you're experiencing an issue with a Microsoft 365 service and you don’t see it listed on the **Service health** page, tell us about it by selecting **Report an issue**, and completing the short form. We’ll look at related data and reports from other organizations to see how widespread the issue is, and if it originated with our service. If it did, we’ll add it as a new incident or advisory on the **Service health** page, where you can track its resolution. The **Reported Issues** page will show all issues your tenant has reported from this form and the status.  

Reviewer: ```suggestion
If you're experiencing an issue with a Microsoft 365 service and you don’t see it listed on the **Service health** page, tell us about it by selecting **Report an issue**, and completing the short form. We’ll look at related data and reports from other organizations to see how widespread the issue is, and if it originated with our service. If it did, we’ll add it as a new incident or advisory on the **Service health** page, where you can track its resolution. The **Reported Issues** page will show all issues your tenant has reported from this form and the status.
```

> To customize your view of which services show up on the dashboard, select **Preferences** > **Custom view**,  and clear the check boxes for the services you want to filter out of your Service health dashboard view. Make sure that the check box is selected for each service that you want to monitor.

Reviewer: ```suggestion
To customize your view of which services show up on the dashboard, select **Preferences** > **Custom view**,  and clear the checkboxes for the services you want to filter out of your Service health dashboard view. Make sure that the checkbox is selected for each service that you want to monitor.
```

- Whether you want notifications for incidents or advisories
- The services for which you want notification

> You can also subscribe to email notifications for individual events instead of every event for a service. To do so click on the active issue you want to recieve email notification updates for, click on **Manage notifications for this issue**, and then specify: 

Reviewer: ```suggestion
You can also subscribe to email notifications for individual events instead of every event for a service. To do so, select the active issue you want to receive email notification updates for, select **Manage notifications for this issue**, and then specify: 
```

|**False positive** | After a detailed investigation, we’ve confirmed the service is healthy and operating as designed. No impact to the service was observed or the cause of the incident originated outside of the service. |
|**Post-incident report published** | We’ve published a Post Incident Report for a specific issue that includes root cause information and next steps to ensure a similar issue doesn’t reoccur. |

### Message Post Types

| Type | Definition |
|:-----|:-----|
|**Quick Update** | Short and frequent incremental updates for broadly impacting incidents, available to all customers. |

> |**Additional Details** | These additional posts will provide richer technical and resolution details to offer deeper visibility into the handling of Incidents. This is available for tenants that meet the same requirements outlined for [Exchange Online monitoring](https://docs.microsoft.com/en-us/microsoft-365/enterprise/microsoft-365-exchange-monitoring?view=o365-worldwide#requirements) |

Reviewer: ```suggestion
|**Additional Details** | These additional posts will provide richer technical and resolution details to offer deeper visibility into the handling of incidents. This is available for tenants that meet the same requirements outlined for [Exchange Online monitoring](/microsoft-365/enterprise/microsoft-365-exchange-monitoring?view=o365-worldwide#requirements), |
```

### History

> Service health lets you look at current health status and view the history of any service advisories and incidents that have affected your tenant in the past 30 days. To view the past health of all services, select **History** view.

Reviewer: ```suggestion
Service health lets you look at your current health status and view the history of any service advisories and incidents that have affected your tenant in the past 30 days. To view the past health of all services, select **History** view.
```
